test/evergreen.yml

```diff
@@ -5334,3 +5334,43 @@ buildvariants:
     - name: unit-test-nonstandalone
     - name: unit-test-long-nonstandalone
       distros: ubuntu2004-arm64-large
+
+- name: amazon2-arm64
+  display_name: "~ Amazon Linux 2 ARM64"
+  run_on:
+  - amazon2-arm64-small
+  batchtime: 1440 # 24 hours
+  expansions:
+    configure_env_vars:
+      PATH=/opt/mongodbtoolchain/v3/bin:$PATH
+    test_env_vars:
+      WT_TOPDIR=$(git rev-parse --show-toplevel)
+      WT_BUILDDIR=$WT_TOPDIR/cmake_build
+      LD_LIBRARY_PATH=$WT_BUILDDIR
+    CMAKE_INSTALL_PREFIX: -DCMAKE_INSTALL_PREFIX=$(pwd)/cmake_build/LOCAL_INSTALL
+    python_binary: '/opt/mongodbtoolchain/v3/bin/python3'
+    smp_command: -j $(echo "`grep -c ^processor /proc/cpuinfo` * 2" | bc)
+    cmake_generator: "Unix Makefiles"
+    make_command: make
+  tasks:
```

Comment:
Can we add in a few non-standalone tasks, to simulate the MDB cloud environment? I'd suggest starting with `make-check-nonstandalone` and `unit-test-nonstandalone`.

Reply:
Actually, if non-standalone tasks are needed, the corresponding standalone tasks can be saved. I believe the main purpose of introducing this new builder is to better match the Atlas environment.

Reply:
I forgot about those. I added the same fleet of tests we have for ubuntu2004-arm64, see e9da8eb476cd1fb6899625b2dd0d881bb882a1b2

Reply:
Thanks.